roapi/src/server/postgres.rs

```diff
@@ -60,9 +69,19 @@ impl<H: RoapiContext> Engine for RoapiContextEngine<H> {
         &mut self,
         statement: &Statement,
     ) -> Result<Self::PortalType, ErrorResponse> {
-        let query = statement.to_string();
-        let df = self.ctx.sql_to_df(&query).await.map_err(df_err_to_sql)?;
-        Ok(DataFusionPortal { df })
+        if RoapiContextEngine::<H>::ignored_statement(statement) {
+            Ok(DataFusionPortal {
+                df: self
+                    .ctx
+                    .sql_to_df("SELECT 1 WHERE 1 = 2")
```

Comment:
If I understand correctly, `where 1 = 2` is added to make sure return empty result

Reply:
yeah, it's hacky, if you have better ideas, let me know!